validator/checker.py

```diff
@@ -3,6 +3,7 @@
 https://github.com/rotki/rotki/blob/8f41fcba4732fba4af563c45c34ad9ad288906a2/rotkehlchen/globaldb/updates.py
 """
 
+import logging
```

Comment:
Hey @yabirgb I just noticed something. You are using the actual logging module. For pytest it's better to use 

`test_warnings` as we do for example here:  https://github.com/rotki/rotki/blob/f053a697d688035e3d3f6bce2d16733d02fa10ed/rotkehlchen/tests/unit/test_assets.py#L134-L139